### DISCUSSION TITLE
2022/04/01/hello-world/

### DISCUSSION THREAD

**Discussion by giscus** [nawoto/nawo.to#29] [Announcements]
# 2022/04/01/hello-world/

雑多な文章を書きたいなっと思ったので、放置していた自分のサイトをGatsbyで作り直してみた。
以前に、Hugo を使ってみたけど、自分のスキルだと細かい部分まで手をいれられなかったで Gatsby…

https://nawo.to/2022/04/01/hello-world/#comments

**Comment by nawoto** [comment]
Hi!!
(test)
